### DISCUSSION TITLE
Duration 0

### DISCUSSION THREAD

**Discussion by ctpntn** [kimai/kimai#2315] [Questions & Answers]
At the moment my user can set duration at 0. How to inhibit this possibility?

**Comment by kevinpapst** [comment]
You can configure rounding, but you can't prevent zero length records.
I am seriously interested what is your thought behind that question!
Why would you prevent that? Maybe it is some kind of reminder.

**Reply by ctpntn** [reply]
Users have to use this tools in order to let me know the time they spent on doing something. What is the meaning of reporting at 0?

**Comment by HeinzWuert** [comment]
This is a time booking tool and not a kind of reminder.
When proofing invoices for admins it's not possible to identify if this is an error. 
As I wrote in the other FR, make it optional or display these actions in another color during the booking to prevent users in producing errors.

**Reply by ctpntn** [reply]
I'm sorry, probably I can't explain myself. Precisely because it is a time booking I don't understand why giving users the option to enter a time 0.

**Comment by kevinpapst** [comment]
Thanks for the explanations.
I tend to think "why not leave it up to the user". But then I don't manage systems for a larger user base. 
So if this is a real world problem, totally accepted. 
@ctpntn please leave your 👍 vote in #2342 - as this is a discussion, I will not keep track of it

**Comment by HeinzWuert** [comment]
Don't  no why u support a reminder in this way? If somebody needs this he may buy your tasks modul, or do u think I'm wrong?

**Reply by kevinpapst** [reply]
I like the way you are thinking 😁

**Comment by HeinzWuert** [comment]
Workaround to prevent 0 Minutes, possible with V1.13

Activate in settings menu > TIMESHEET > Time tracking mode
- (Duration) replaces endtime with a duration  input-field 
and
- Minute selection for From & To to minimum 1 or what ever u want/need
- Minute selection for Duration to minimum 1 or what ever u want/need
SAVE
that;s it

**Reply by BeckeBauer** [reply]
Before I purchased the expenses bundle, I used the 0 minutes time sheets for expenses. So, when I booked for instance travel costs on a customer on 11.2., I created a time sheet for this day with 0 minutes for this customer and added the costs...
